lms/djangoapps/certificates/queue.py

```diff
@@ -274,7 +274,10 @@ def add_cert(self, student, course_id, course=None, forced_grade=None, template_
             template_pdf = "certificate-template-{id.org}-{id.course}-verified.pdf".format(id=course_id)
         elif mode_is_verified and not user_is_verified:
             template_pdf = "certificate-template-{id.org}-{id.course}.pdf".format(id=course_id)
```

Comment:
It seems odd to have an honor track still active when we only offer audit and verified tracks at this point.  Are we certain we need to generate certs in 'honor' mode?